test-lib-remote-openshift.sh

```diff
@@ -98,16 +98,11 @@ function ct_os_import_image_ocp4() {
   local image_name="${1}"; shift
   local imagestream=${1:-$image_name:latest}
   local namespace
+  local image_name_no_namespace=${image_name##*/}
+  local image_tagged="${image_name_no_namespace%:*}:${VERSION}"
```

Comment:
I don't understand why those variables are added, they seem to be not used in the function.